inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -130,6 +144,10 @@ void MKLDNNConvolutionNode::getSupportedDescriptors() {
     if (!descs.empty())
         return;
 
+    // winograd support only constant weights and bias
+    isWino &= getParentEdgeAt(1)->getParent()->isConstant() && getParentEdgeAt(1)->getParent()->getType() == Input &&
```

Comment:
I think we should explicitly disable winograd for int8. It also interesting will the behavior correct in case Sum was fused during transformation stage.

Reply:
int8 winograd disabled now
do you mean full shape sum? if yes, it's work correct we have test for this.
 ```getOriginalInputsNumber()``` return original inputs number based on ```originalInputPrecisions```, we update ```originalInputPrecisions``` only in ```FuseConvolutionAndBias``` and ```FuseMultiplyAndAdd```